doc/manual/trmacros.txt

```diff
@@ -50,7 +54,7 @@ blindly:
 .. code-block:: nim
   template mulIsCommutative{`*`(a, b)}(a, b: int): int = b*a
   
-What optimizers really need to do is a *canonicalization*:
+What optimizers really need to do is a *canonization*:
```

Comment:
Googled. Hehe, yeah, that was quite a different thing. I'll correct it to the original word. We could add a {. saint .} pragma instead for the other use ;)